2020/test/MineralDetectionTask.java

```diff
@@ -119,11 +119,11 @@ public void init(Telemetry telemetry, HardwareMap hardwareMap)
     {
         initVuforia(hardwareMap);
 
-        if (ClassFactory.getInstance().canCreateTFObjectDetector()) {
+        //if (ClassFactory.getInstance().canCreateTFObjectDetector()) {
             initTfod(hardwareMap);
-        } else {
-            telemetry.addData("Sorry!", "This device is not compatible with TFOD");
-        }
+        //} else {
+        //    telemetry.addData("Sorry!", "This device is not compatible with TFOD");
+        //}
```

Comment:
There shouldn't be any reason to comment this out.  Can you please describe what prompted this?

Reply:
When we'd finished writing code for the wobble goal, we ran a build process. All the build errors were those lines--build gave a compilation error unless we commented it out. Cindy said that those lines aren't necessary, so we just commented them out.

Reply:
Right, so that method was removed, by an overzealous tech team member when he stripped code for KitKat support out of the SDK in the early summer of 2020.

Note that he should have marked it as deprecated so that he didn't break team code, but that didn't happen.

Since that method was specifically designed to alert users when they were using a kitkat phone, and since that method no longer exists in the SDK, please remove these lines entirely, instead of just commenting them out.  If you remove them in your local workspace, and then do a commit on this same branch, and then push to the same branch then the PR will update automagically and I'll approve and merge.

Reply:
Should be done now.